From eea6634d560b051964bf8ebb6c8d51c321e03d20 Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz@ettus.com>
Date: Sun, 15 Dec 2013 18:03:32 +0100
Subject: [PATCH 10/39] boost: Bump required boost version to 1.41.

* The addition of boost::property_tree entails dependency on 1.41
  or higher.

* Old versions have been removed from the boost_ADDITIONAL_VERSIONS
  variable.

* Removed future untested (possibly API breaking) versions of
  boost beyond currently released boost stable of 1.55.

Signed-off-by: Moritz Fischer <moritz@ettus.com>
---
 host/CMakeLists.txt |   12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 4ecdbd5..28ad6b9 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -154,15 +154,11 @@ IF(MSVC)
 ENDIF(MSVC)
 
 SET(Boost_ADDITIONAL_VERSIONS
-    "1.35.0" "1.35" "1.36.0" "1.36" "1.37.0" "1.37" "1.38.0" "1.38" "1.39.0" "1.39"
-    "1.40.0" "1.40" "1.41.0" "1.41" "1.42.0" "1.42" "1.43.0" "1.43" "1.44.0" "1.44"
-    "1.45.0" "1.45" "1.46.0" "1.46" "1.47.0" "1.47" "1.48.0" "1.48" "1.49.0" "1.49"
-    "1.50.0" "1.50" "1.51.0" "1.51" "1.52.0" "1.52" "1.53.0" "1.53" "1.54.0" "1.54"
-    "1.55.0" "1.55" "1.56.0" "1.56" "1.57.0" "1.57" "1.58.0" "1.58" "1.59.0" "1.59"
-    "1.60.0" "1.60" "1.61.0" "1.61" "1.62.0" "1.62" "1.63.0" "1.63" "1.64.0" "1.64"
-    "1.65.0" "1.65" "1.66.0" "1.66" "1.67.0" "1.67" "1.68.0" "1.68" "1.69.0" "1.69"
+    "1.41.0" "1.41" "1.42.0" "1.42" "1.43.0" "1.43" "1.44.0" "1.44" "1.45.0" "1.45"
+    "1.46.0" "1.46" "1.47.0" "1.47" "1.48.0" "1.48" "1.48.0" "1.49" "1.50.0" "1.50"
+    "1.51.0" "1.51" "1.52.0" "1.52" "1.53.0" "1.53" "1.54.0" "1.54" "1.55.0" "1.55"
 )
-FIND_PACKAGE(Boost 1.36 COMPONENTS ${BOOST_REQUIRED_COMPONENTS})
+FIND_PACKAGE(Boost 1.41 COMPONENTS ${BOOST_REQUIRED_COMPONENTS})
 
 INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
 LINK_DIRECTORIES(${Boost_LIBRARY_DIRS})
-- 
1.7.10.4

